Makefile

```diff
@@ -206,6 +207,18 @@ include graphics.mk
 chao_garden/mb_chao_garden.gba.lz: chao_garden/mb_chao_garden.gba 
 	$(GFX) $< $@ -search 1
 
+%/interactables.bin: %/interactables.csv
+	$(EPOS) $< $@ -entities INTERACTABLES -header "./include/constants/interactables.h"
+
+%/itemboxes.bin: %/itemboxes.csv
```

Comment:
Do you need the `%/` here?

Reply:
Well, the slash is optional, but without the `%` it doesn't build.